src/setupTests.ts

```diff
@@ -0,0 +1,9 @@
+// setup file
+import '@testing-library/jest-dom';
+import { createIntl } from 'react-intl';
+import messages from './translations/translations.json';
+
+export const intl = createIntl({
+  locale: 'en',
+  messages: messages.en,
+});
```

Comment:
Not sure if there’s a simple way to do something like `renderWithIntl(<MyComponent />)` which won’t cause type errors. For now, testing components that rely on `react-intl` requires manually passing the `intl` prop:

```tsx
import { intl } from '../setupTests';

// in a test case:
render(<MyComponent intl={intl} />
```